doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,32 @@
-=======================
-Release Build Creation
-=======================
+===========================
+Compile & Build Cortx Stack
+===========================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
-***************
-Procedure
-***************
+*****************************************
+Specifications to setup Virtual Machine
+*****************************************
 
-#. Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system.
+- Single-Node VM deployment
 
-   - You can use a Virtual Machine (VM) also.
+  - Please create VM(s) with at least 2 cores (vCPU) and 4GB of RAM.
+  - Ensure the VM is created with minimum of total 2 attached raw disks per node
+
+    - Data Disks - Min 1 Disks per node (Capacity 10G+)
+    - Metadata Disks - Min 1 Disks per node (Capacity - 10% of total Data Disk Size)
+  - Ensure the VM is created with minimum 3 network Interfaces
+  - Ensure system have valid hostname and they are accessible (i.e. ping)
+  - Ensure IP’s have assigned to all NICs e.g. eth33, eth34 and eth35
```

Comment:
Can we add a note that your NIC's may have different names?  
  
Also, I don't think 10GB is enough space for the disk. I had an issue where I ran out of storage when I used disks \<50GB for the data disk. The metadata disks can be small but the main disk needs to be bigger than this.  
  
Also, you can do this with 1GB of RAM and 1 core, I think it is worth making a note of this.

Reply:
done